packages/components/src/Icon/subcomponents/SVG/SVG.tsx

```diff
@@ -15,6 +15,7 @@ type DecorativeIcon = {
 
 export type BaseSVGProps = {
   inheritSize?: boolean
+  shouldFlipOnRtl?: boolean
```

Comment:
Probably don't want to expose this as a prop, since this might imply that any icon can be flipped for RTL. 

Maybe we give the icons a unique class name and target them that way

Reply:
Ah woops, sorry that's actually just part of the exploration I did when I was looking at components - I'll remove that from the PR 👍